Client/RecommendationRequestClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Client;
+
+interface RecommendationRequestClient
+{
+    /**
+     * @param int $userId
+     * @param int $scenarioId
+     * @param int $locationId
+     * @param int $limit
+     * @return string|null JSON response
```

Comment:
Are you sure that returning JSON at this step is a good idea ?

Since the interface is meant to be used in controllers and scripts, it should rather return some kind of "RecommendedContent" object (suggested name, but maybe it's actually a Location or something).

Could it maybe return a "real" API Value Object ? Any particular reason why you made it use a custom struct instead of an existing Object from the API ?